### PR TITLE
Th2 1440

### PR DESCRIPTION
**Pull request by IvanChadin**

### CODE REVIEW

File: th2_common/schema/message/configuration/queue_configuration.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ class QueueConfiguration(Configuration):
 
     def __init__(self, name: str, queue: str, exchange: str, attributes: list, filters: list,
```
Comment (RaevskiySV) [inline th2_common/schema/message/configuration/queue_configuration.py:22]: Maybe we should rename name parameter to routing_key as well?

File: th2_common/schema/message/impl/rabbitmq/parsed/rabbit_parsed_batch_subscriber.py [hunk 2]
```diff
@@ -50,7 +50,17 @@ def extract_metadata(self, message) -> Metadata:
...
+    @staticmethod
+    def value_from_bytes(body):
+        message_group_batch = MessageGroupBatch()
+        message_group_batch.ParseFromString(body)
+
+        message_batches = []
+        for message_group in message_group_batch.groups:
+            messages = []
+            for any_message in message_group.messages:
+                any_message.HasField('message')
```
Comment (RaevskiySV) [inline th2_common/schema/message/impl/rabbitmq/parsed/rabbit_parsed_batch_subscriber.py:62]: Missing if?

File: th2_common/schema/message/impl/rabbitmq/raw/rabbit_raw_batch_subscriber.py [hunk 3]
```diff
@@ -45,10 +45,20 @@ def extract_count_from(self, message: RawMessageBatch):
...
+    @staticmethod
+    def value_from_bytes(body):
+        message_group_batch = MessageGroupBatch()
+        message_group_batch.ParseFromString(body)
+
+        message_batches = []
+        for message_group in message_group_batch.groups:
+            messages = []
+            for any_message in message_group.messages:
+                any_message.HasField('raw_message')
```
Comment (RaevskiySV) [inline th2_common/schema/message/impl/rabbitmq/raw/rabbit_raw_batch_subscriber.py:57]: Missing if?

**Review by RaevskiySV — APPROVED** [review]

File: th2_common/schema/message/impl/rabbitmq/parsed/rabbit_parsed_batch_subscriber.py (hunk 2 above)
Comment (IvanChadin) [reply th2_common/schema/message/impl/rabbitmq/parsed/rabbit_parsed_batch_subscriber.py:62]: No, exception is thrown here

File: th2_common/schema/message/impl/rabbitmq/raw/rabbit_raw_batch_subscriber.py (hunk 3 above)
Comment (IvanChadin) [reply th2_common/schema/message/impl/rabbitmq/raw/rabbit_raw_batch_subscriber.py:57]: No, exception is thrown here

**Pull request merged**